src/postgres-util/src/tunnel.rs

```diff
@@ -284,3 +298,10 @@ impl Config {
         self.inner.get_dbname()
     }
 }
+
+fn host_mut(config: &mut tokio_postgres::Config) -> Result<&mut String, PostgresError> {
+    match config.get_hosts_mut() {
+        [Host::Tcp(host)] => Ok(host),
+        _ => bail_generic!("only TCP connections to a single PostgreSQL server are supported"),
+    }
+}
```

Comment:
I feel like this should go next to `fn address` as a method.

Reply:
Ah, sorry, good point. In that case it seems like this would be a little cleaner if inlined? AFIACT it's only called in one place, and `*host_mut(&mut postgres_config)?` is really weird to read.